samples/04_gis_analysts_data_scientists/Training a Deep Learning model, for using it in 3D basemap Solutions..ipynb

```diff
@@ -0,0 +1,383 @@
+{
```

Comment:
<ol><li>Convert the dataset into&nbsp;<code>.las</code>&nbsp;format.<em>&nbsp;(Refer to ArcGIS Pro's&nbsp;</em><a href="https://pro.arcgis.com/en/pro-app/tool-reference/conversion/convert-las.htm" target="_blank"><em>'Convert LAS'</em></a><em>&nbsp;GP tool.)</em></li><li>Split all the&nbsp;<code>.las</code>&nbsp;files into three unique sets, one for training, one for validation, and one for testing. There is no fixed rule, but generally, the validation data for point cloud in&nbsp;<code>.las</code>&nbsp;format should be at least 1 % (by size) of the total data available, with appropriate diversity within the validation dataset.<em>&nbsp;(For ease in splitting the big&nbsp;</em><code><em>.las</em></code><em>&nbsp;files into the appropriate ratios, ArcGIS Pro's&nbsp;</em><a href="https://pro.arcgis.com/en/pro-app/tool-reference/3d-analyst/tile-las.htm" target="_blank"><em>'Tile LAS'</em></a><em>&nbsp;GP tool can be used.)</em></li><li>Reassign classcodes if needed. Consider the 'object of interest' or any requirement related to the workflow of the 3D Basemap solution.<em>&nbsp;(Refer to ArcGIS Pro's&nbsp;</em><a href="https://pro.arcgis.com/en/pro-app/tool-reference/3d-analyst/change-las-class-codes.htm" target="_blank"><em>'Change LAS Class Codes'</em></a><em>&nbsp;GP tool.)</em></li></ol><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/763/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
corrected accordingly' Thanks.